### ISSUE TITLE
Overflow encountered in exp from bbox_transform.py

### ISSUE THREAD

**Issue by karaspd** [playerkk/face-py-faster-rcnn#19]
I followed all the instruction, the only thing that I changed was the caffe version (since I am using cuda 8.0 cudnn 5).
when I train the network,it broken down after print one log:

I1130 15:33:11.263000   990 solver.cpp:218] Iteration 0 (0 iter/s, 2.04132s/20 iters), loss = 3.34715
I1130 15:33:11.263075   990 solver.cpp:237]     Train net output #0: accuracy = 0.515625
I1130 15:33:11.263094   990 solver.cpp:237]     Train net output #1: loss_bbox = 0.0558917 (* 1 = 0.0558917 loss)
I1130 15:33:11.263106   990 solver.cpp:237]     Train net output #2: loss_cls = 0.895517 (* 1 = 0.895517 loss)
I1130 15:33:11.263116   990 solver.cpp:237]     Train net output #3: rpn_cls_loss = 0.709608 (* 1 = 0.709608 loss)
I1130 15:33:11.263128   990 solver.cpp:237]     Train net output #4: rpn_loss_bbox = 1.69873 (* 1 = 1.69873 loss)
I1130 15:33:11.263144   990 sgd_solver.cpp:105] Iteration 0, lr = 0.001
/opt/caffe/tools/../lib/fast_rcnn/bbox_transform.py:48: RuntimeWarning: overflow encountered in exp
  pred_w = np.exp(dw) * widths[:, np.newaxis]
/opt/caffe/tools/../lib/fast_rcnn/bbox_transform.py:49: RuntimeWarning: overflow encountered in exp
  pred_h = np.exp(dh) * heights[:, np.newaxis]
/opt/caffe/tools/../lib/rpn/proposal_layer.py:175: RuntimeWarning: invalid value encountered in greater_equal
  keep = np.where((ws >= min_size) & (hs >= min_size))[0]
I1130 15:33:24.214957   990 solver.cpp:218] Iteration 20 (1.54418 iter/s, 12.9518s/20 iters), loss = nan
I1130 15:33:24.215034   990 solver.cpp:237]     Train net output #0: accuracy = 1
I1130 15:33:24.215051   990 solver.cpp:237]     Train net output #1: loss_bbox = nan (* 1 = nan loss)
I1130 15:33:24.215062   990 solver.cpp:237]     Train net output #2: loss_cls = 87.3365 (* 1 = 87.3365 loss)
I1130 15:33:24.215075   990 solver.cpp:237]     Train net output #3: rpn_cls_loss = 87.3365 (* 1 = 87.3365 loss)
I1130 15:33:24.215085   990 solver.cpp:237]     Train net output #4: rpn_loss_bbox = nan (* 1 = nan loss)
I1130 15:33:24.215096   990 sgd_solver.cpp:105] Iteration 20, lr = 0.001

**Comment by playerkk** [comment]
It's hard to say what happened. You might first want to check your ground-truth annotations.

**Comment by karaspd** [comment]
@playerkk thanks for your respond. My guess is something wrong in the Wider dataset annotations file, however I am using the same annotations the instruction link provided. I tried same code with pascal voc it looks fine so far. Do you think anything in annotation file has been changed since you worked with it?

**Comment by karaspd** [comment]
@playerkk  It was a problem in my prototxt file. I removed the last layer:
layer {
  name: "accuracy"
  type: "Accuracy"
  bottom: "cls_score"
  bottom: "labels"
  top: "accuracy"
}
